pycam/__init__.py

```diff
@@ -18,7 +18,8 @@
 along with PyCAM.  If not, see <http://www.gnu.org/licenses/>.
 """
 
-VERSION = "0.6.1-dev"
+from pycam.version import VERSION  # noqa F401
```

Comment:
I felt uncomfortable with your approach of creating a non-versioned file and now I went as far as overwriting a versioned one. Hm - I need to think about this again.